airbyte-integrations/connectors/source-mailchimp/unit_tests/utils.py

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+from typing import Any, MutableMapping
+
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources.streams import Stream
+
+
+def read_incremental(stream_instance: Stream, stream_state: MutableMapping[str, Any]):
```

Comment:
Can we make one util function instead of two by passing all the necessary params inside and move it to the file where it is used? Because these functions are used just once and I'd prefer to see them side by side with the tests to understand what's going on inside. What do you think?

Reply:
never mind, I see it's a common case for most of connectors